appendix.rst

```diff
@@ -164,6 +164,11 @@ below with their respective functionality.
 #. **{ENVPREFIX}_ENCRYPTION_PASSPHRASE**: Used to specify the plaintext
    passphrase to encrypt the container.
 
+#. **{ENVPREFIX}_ENCRYPTION_PEM_DATA**: If it contains data from a public PEM 
+   file Apptainer can use those data to encrypt a container. If it contains 
```

Comment:
Add a comma after "file" (like in the next sentence), and replace the two `Apptainer` strings with `{Project}` to follow the documentation convention.